README.md

```diff
@@ -40,27 +40,21 @@ and Windows Assessment and Deployment Kit (ADK)
 the windows-curtin-hooks and WindowsUpdates git submodules are required.<br/>
 Run `git submodule update --init` to retrieve them
 * Import the WinImageBuilder.psm1 module
-* Use the New-WindowsCloudImage or New-WindowsOnlineCloudImage methods with <br/> the appropriate configuration options
+* Use the New-WindowsCloudImage or New-WindowsOnlineCloudImage methods with <br/> the appropriate configuration file
 
 ### PowerShell image generation example for OpenStack KVM (host requires Hyper-V enabled)
 ```powershell
 git clone https://github.com/cloudbase/windows-openstack-imaging-tools.git
 pushd windows-openstack-imaging-tools
 Import-Module .\WinImageBuilder.psm1
+Import Config.psm1
+Create a config.ini file with the necessary parameters for the function you want to use. The parameters can be found in the Config.psm1 module
```

Comment:
add the actual working command here:
$configFilePath = "config.ini"
New-WindowsImageConfig ...